### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by fithamlak**
`project-1`- create animals table with the following functionalities implemented

-  Find all animals whose name ends in "mon". 
ends-with-mon
-  List the name of all animals born between 2016 and 2019.

-  List the name of all animals that are neutered and have less than 3 escape attempts.
neutered-and-lessthan3-updated
-  List the date of birth of all animals named either "Agumon" or "Pikachu".
Agumen-or-Pikachu
-  List name and escape attempts of animals that weigh more than 10.5kg
more-than-10 5kg
-  Find all animals that are neutered.
neutered
-  Find all animals not named Gabumon.
not-Gabumon
-  Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,13 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attepts, neutered, weight_kg)
+VALUES ('Agumen', 'Feb 03, 2020', 0, true, 10.23 );
```
Comment (Davidosky007) [inline data.sql:4]: * Kindly correct the animal name `Agumon`  not `Agumen` as stated in the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md)

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where name like '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = TRUE and escape_attepts < 3;
+SELECT date_of_birth from animals WHERE name IN ('Agumen', 'Pikachu'); 
+SELECT name, escape_attepts from animals WHERE weight_kg > 10.5;
+SELECT *  from animals WHERE name != 'Gabumon';
```
Comment (Davidosky007) [inline queries.sql:8]: * Kindly write the query for this `Find all animals that are neutered.` as its missing [Check requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=Find%20all%20animals%20that%20are%20neutered.)

File: README.md [hunk 3]
```diff
@@ -280,12 +160,6 @@ I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
-
-## 📝 License <a name="license"></a>
-
-This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (Davidosky007) [inline README.md:152]: * Kindly add the MIT link

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @fithamlak 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (fithamlak) [reply queries.sql:8]: I have already done it, it is in the six query and image dear

**Comment by arslanbisharat** [comment]
Hi @fithamlak,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fithamlak** [comment]
thank you

**Pull request merged**